README.md

```diff
@@ -47,18 +55,26 @@ If you're using Docker, you can use the [official Dgraph image](https://hub.dock
 docker pull dgraph/dgraph:latest
 ```
 
+For more information on a variety Docker deployment methods including Docker Compose and Kubernetes, see the [docs](https://dgraph.io/docs/deploy/single-host-setup/#run-using-docker).
+
+## Run a Quick Standalone Cluster
+
+```
+docker run --rm -it -p 8080:8080 -p 9080:9080 -p 8000:8000 -v ~/dgraph:/dgraph dgraph/standalone:latest
```

Comment:
`-p 8000:8000` are we adding Ratel in the standalone dockerfile?

Reply:
Doesn't seem like it is. I just logged into a running standalone image, and dgraph-ratel is not in the image

Reply:
@matthewmcneely the `--rm` option, will cleanup on container exit. Can we remove the `--rm` here?